csharp/extractor/Semmle.Extraction.CIL/Tuples.cs

```diff
@@ -200,6 +200,9 @@ internal static class Tuples
         internal static Tuple locations_default(ISourceLocation label, IFile file, int startLine, int startCol, int endLine, int endCol) =>
             new Tuple("locations_default", label, file, startLine, startCol, endLine, endCol);
 
+        internal static Tuple metadata_handle(IEntity entity, Assembly assembly, int handleValue) =>
```

Comment:
How about changing the type of `handleValue` to `Handle`, and then call `handleValue.GetHashCode()` here rather than at all the call sites?

Reply:
I didn't do that because I didn't want to put population logic into the `Tuples` methods.

Reply:
OK.